src/server/api.ts

```diff
@@ -157,4 +163,33 @@ router.post("/translate", async (req, res) => {
   }
 });
 
+router.post("/invoke-model", async (req, res) => {
+  const client = new BedrockRuntimeClient({ region: "eu-central-1" }); //As of now this is the closest region with the service
+  const modelId = "anthropic.claude-3-haiku-20240307-v1:0";
```

Comment:
Burde dette vore ein secret?

Reply:
De er ikke veldig hemmelig, men vi kan godt sett det opp som environmentvariabler hvis det er ønskelig

Reply:
Nei, om dette er det det heiter og ikkje er noko som er unikt for oss er det ikkje farlig.